testing/go/functions_test.go

```diff
@@ -960,6 +960,13 @@ func TestSystemInformationFunctions(t *testing.T) {
 		{
 			Name: "obj_description",
 			Assertions: []ScriptTestAssertion{
+				{
+					Query:            `SELECT obj_description(100);`,
+					ExpectedColNames: []string{"obj_description"},
+					Expected: []sql.Row{
+						{""},
```

Comment:
So this result should actually be `"database's default collation"`, but of course it's not implemented yet. Instead, I think it'd be better to use `SELECT obj_description(1003);`, which actually returns an empty string as the result. That way, the test is actually correct, as we may not know later that this test is here to ensure functionality, not validate results.